ergotree-ir/src/types/sglobal.rs

```diff
@@ -61,3 +64,18 @@ lazy_static! {
     pub static ref XOR_METHOD: SMethod = SMethod::new(STypeCompanion::Global, XOR_METHOD_DESC.clone(),);
 
 }
+
+lazy_static! {
+    static ref FROM_BIGENDIAN_BYTES_METHOD_DESC: SMethodDesc = SMethodDesc {
+        method_id: FROM_BIGENDIAN_BYTES_METHOD_ID,
+        name: "fromBigEndianBytes",
+        tpe: SFunc {
+            t_dom: vec![SType::SGlobal, SType::SColl(SType::SByte.into())],
+            t_range:SType::STypeVar(STypeVar::t()).into(),
+            tpe_params: vec![STypeParam::param_t()],
```

Comment:
I don't think this is the right way to do this. Since the output type of fromBigEndianBytes can't be inferred from input, the output type needs to be explicitly serialized/deserialized, so add it to explicit_type_args list, see: https://github.com/ergoplatform/sigma-rust/blob/f6048c1e4e4813af9308c484b3ab7fe904ce7005/ergotree-ir/src/types/sbox.rs#L60

Also, can you add roundtrip tests for MethodCalls using this new method? You can skip versioning for now as versioning semantics are still not final in sigmastate-interpreter

Reply:
Thanks for the review. I'll add explicit type args for proper type inference and include proptest roundtrip tests for MethodCall.